docs/run/advanced-runtime-options.mdx

```diff
@@ -140,15 +234,33 @@ estimator = Estimator(session=backend, options=options)
 
 For more information and a complete list of advanced transpilation options, see the Advanced transpilation options table in the
 [Runtime compilation topic](configure-runtime-compilation#transpilation-table).
+  </TabItem>
+</Tabs>
 
 ### Error mitigation
 
 
 You might want to leverage different error mitigation methods and see how these affect the performance of your
-algorithm. These can also be set through the `resilience_level` option. The method selected for each level is
-different for `Sampler` and `Estimator`. You can find more information in the
+algorithm.  These methods can be set through the `resilience_level` option.  For more information about error mitigation, see the
 [Configure error mitigation topic](configure-error-mitigation).
 
+<Tabs>
+  <TabItem value="PrimV2" label="V2 Estimator">
+With V2 Estimator, you can set resilience levels 0-2 and you can also turn on and off various error mitigation settings for fine tuning. 
+
+```python
+estimator = Estimator(backend=backend)
+
+estimator.options.resilience_level = 2
+estimator.options.dynamical_decoupling.enable = True
+estimator.options.dynamical_decoupling.sequence_type = "XpXm"
```

Comment:
Not that it matters much, but since the topic is error mitigation, let's use an error mit example (DD is considered error suppression).
```suggestion
estimator.options.zne_mitigation = True
estimator.options.zne.noise_factors = [1, 3, 5]
```